### PR TITLE
Latest experimental

### PR DESCRIPTION
**Pull request by jcabrero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/NillionNetwork/nada-numpy/test.yml?style=for-the-badge&logo=python&logoColor=white&link=https%3A%2F%2Fgithub.com%2FNillionNetwork%2Fnada-numpy%2Factions%2Fworkflows%2Ftest.yml&link=https%3A%2F%2Fgithub.com%2FNillionNetwork%2Fnada-numpy%2Factions%2Fworkflows%2Ftest.yml)
 ![GitHub Release](https://img.shields.io/github/v/release/NillionNetwork/nada-numpy?sort=date&display_name=release&style=for-the-badge&logo=dependabot&label=LATEST%20RELEASE&color=0000FE&link=https%3A%2F%2Fpypi.org%2Fproject%2Fnada-numpy&link=https%3A%2F%2Fpypi.org%2Fproject%2Fnada-numpy)
 
-
+# WARNING: THIS VERSION DOES NOT INSTALL `nada_dsl` and `py_nillion_client` by default as it needs to be installed by `nilup`.
```
Comment (mathias-nillion) [inline README.md:7]: nit: let's add `nilup install [VERSION_NAME] --nada-dsl --python-client` as the command to use?

File: pyproject.toml [hunk 2]
```diff
@@ -1,17 +1,16 @@
 [tool.poetry]
 name = "nada-numpy"
-version = "0.1.3"
+version = "0.3.0"
```
Comment (mathias-nillion) [inline pyproject.toml:3]: why are we leapfrogging 0.2.x?

**Review by mathias-nillion — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (jcabrero) [reply README.md:7]: Good point, I need to remove that line as it is already included automatically. This was only necessary for the other part.

**Pull request merged**